apps/browser/src/autofill/services/autofill.service.ts

```diff
@@ -1287,28 +1328,173 @@ export default class AutofillService implements AutofillServiceInterface {
    * Used when handling autofill on credit card fields. Determines whether
    * the field has an attribute that matches the given value.
    * @param {AutofillField} field
-   * @param {string} containsVal
+   * @param {string} containsValue
    * @returns {boolean}
    * @private
    */
-  private fieldAttrsContain(field: AutofillField, containsVal: string): boolean {
+  private fieldAttrsContain(field: AutofillField, containsValue: string): boolean {
     if (!field) {
       return false;
     }
 
-    let doesContain = false;
-    CreditCardAutoFillConstants.CardAttributesExtended.forEach((attr) => {
-      // eslint-disable-next-line
-      if (doesContain || !field.hasOwnProperty(attr) || !field[attr]) {
+    let doesContainValue = false;
+    CreditCardAutoFillConstants.CardAttributesExtended.forEach((attributeName) => {
+      // eslint-disable-next-line no-prototype-builtins
+      if (doesContainValue || !field.hasOwnProperty(attributeName) || !field[attributeName]) {
```

Comment:
```suggestion
      if (doesContainValue || !field[attributeName]) {
```

Nit - Since we're already touching this piece of code, we should be able to remove this builtin call on the `field` object safely.